contracts/examples/nft-minter/src/nft_module.rs

```diff
@@ -169,15 +169,15 @@ pub trait NftModule {
             sc_panic!("Attributes encode error: {}", err.message_bytes());
         }
 
-        let attributes_hash: ManagedByteArray<Self::Api, 32> =
-            self.crypto().sha256(&serialized_attributes);
+        let attributes_sha256 = self.crypto().sha256_managed::<SHA256_HASH_DATA_BUFFER_LEN>(&serialized_attributes);
```

Comment:
This should be a larger number here, like 1000.
It is not a standard constant, it is something that depends on the input. For this contract, 1000 is probably reasonable.